### PR TITLE
[SW-636] Improve multisig performance

### PR DESCRIPTION
**Pull request by CedrikNikita**
Table of dry runs called every 7 seconds

**with additional info** means that some pages demand additional info (`MultisigDetails`, `MultisigProposalDetails`), but we will have one more dry-run and only for active account
|Multisig info|Before|After (with additional info)|After (without additional info)|
|---|---|---|---|
|1 visible vault|**3** dry-runs | **2** dry-runs|**1** dry-run|
|10 visible vaults|**30** dry-runs | **11** dry-runs|**10** dry-runs|

### CODE REVIEW

**Review by ifaouibadi — APPROVED** [review]

**Comment by subhod-i** [comment]
@CedrikNikita Did you re-request for review without any changes?

**Review by subhod-i — APPROVED** [review]
LGTM considering MVP as goal.

**Review by pstachel — APPROVED** [review]

**Review by konradodo — APPROVED** [review]

File: src/popup/pages/MultisigProposalDetails.vue [hunk 1]
```diff
@@ -440,9 +443,12 @@ export default defineComponent({
     onMounted(async () => {
       if (activeMultisigAccount.value) {
         getTransactionDetails();
+        fetchAdditionalInfo();
       }
     });
 
+    onBeforeUnmount(() => stopFetchingAdditionalInfo());
```
Comment (konradodo) [inline src/popup/pages/MultisigProposalDetails.vue:450]:
```suggestion
    onBeforeUnmount(stopFetchingAdditionalInfo);
```

File: src/popup/pages/MultisigDetails.vue [hunk 2]
```diff
@@ -107,6 +111,10 @@ export default defineComponent({
       consensusLabel,
     } = activeMultisigAccount.value || {} as IMultisigAccount;
 
+    onMounted(() => fetchAdditionalInfo());
+
+    onBeforeUnmount(() => stopFetchingAdditionalInfo());
```
Comment (konradodo) [inline src/popup/pages/MultisigDetails.vue:116]:
```suggestion
    onMounted(fetchAdditionalInfo);

    onBeforeUnmount(stopFetchingAdditionalInfo);
```

**Comment by Liubov-crypto** [comment]
LGTM

**Pull request merged**